release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,222 @@
+k6 `v1.0.0-rc1` is here 🎉!
+
+This release marks a special, non-conventional milestone in the k6 software life-cycle, serving as a preview of the upcoming version 1.0.0.
+
+The purpose of this release is to give the community a chance to test the new version, identify any potential issues, and test migrations of any parts affected by breaking changes. If you encounter any problems or have trouble with the migration, we encourage you to report them by creating an issue. Your feedback will help improve the final release. If no critical issues are reported, we plan to release the final v1.0.0 version in a couple of weeks.
```

Comment:
Just to confirm, I thought the final v1.0.0 version was going to be released in May. Is this right that it would be in a couple of weeks? 😱

Reply:
@heitortsergent I believe it's a good catch :sweat_smile: The actual v1.0.0 is going to be in May, we need though to set the date in https://github.com/grafana/k6/milestone/48

Reply:
```suggestion
The purpose of this release is to give the community a chance to test the new version, identify any potential issues, and test migrations of any parts affected by breaking changes. If you encounter any problems or have trouble with the migration, we encourage you to report them by creating an issue. Your feedback will help improve the final release. If no critical issues are reported, we plan to release the final `v1.0.0` within the next month.
```